WORKSPACE

```diff
@@ -81,7 +81,7 @@ list_source_repository(name = "local_bazel_source_list")
 # android_sdk_repository(name = "androidsdk")
 # android_ndk_repository(name = "androidndk")
 
-# In order to run //src/test/shell/bazel:maven_skylark_test, follow the
+# In order to run //src/test/shell/bazel:maven_starlark_test, follow the
```

Comment:
I see some unrelated white space changes. I would rather revert them all, and only include related addition related to java_toolchain_jdk15. Clean up changes should be done in their own PR. Just guessing: you have run buildifier on this files, that also changed the white spaces and shuffled around the attributes.

Reply:
Done